### PR TITLE
Added Feature Contact Form

### PR DESCRIPTION
**Pull request by joshuaoivie**
Dear Code Reviewer,

This pull request contains the following changes

-Added contact form intro
-Added contact form
-Fixed lint errors

Warm Regards,
Joshua Ivie

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -167,7 +167,39 @@ <h3 class="title">Skills</h3>
...
+            <textarea name="message" id="contact-message" cols="30" rows="7" maxlength="500"
+              placeholder="Write your message here" required></textarea>
+          </li>
+          <li>
+            <button type="submit" class="submit-btn">Get in Touch</button>
+          </li>
+        </ul>
+      </form>
+    </div>
+  </footer>
```
Comment (mateo951) [inline index.html:202]: There is a white space on the right side of your about and form section that should not be there 🧐

Captura | Captura1

Kindly fix the width for these sections to meet with the proper design 🙌

File: index.html [hunk 2]
```diff
@@ -167,7 +167,39 @@ <h3 class="title">Skills</h3>
...
+      <div class="contact-intro">
+        <div class="title">
+          Contact me
+        </div>
+        <div class="description">
+          If you have an application you are interested in developing, a feature that you need built or a project that
+          needs coding. I'd love to help with it
+        </div>
+      </div>
+      <form action="https://formspree.io/f/xbjwdrql" method="post" class="contact-form">
```
Comment (mateo951) [inline index.html:184]: Nice work applying the formspree functionality 😎

The form has been sent successfully ✔
Captura

File: index.html [hunk 3]
```diff
@@ -167,7 +167,39 @@ <h3 class="title">Skills</h3>
...
+          <li>
+            <input type="email" placeholder="Enter your email" required />
+          </li>
+          <li>
+            <textarea name="message" id="contact-message" cols="30" rows="7" maxlength="500"
+              placeholder="Write your message here" required></textarea>
+          </li>
+          <li>
+            <button type="submit" class="submit-btn">Get in Touch</button>
+          </li>
```
Comment (mateo951) [inline index.html:198]: You used the propper type attribute in all your inputs 👍 Well done 🙌

File: index.html [hunk 4]
```diff
@@ -167,7 +167,39 @@ <h3 class="title">Skills</h3>
...
+      <form action="https://formspree.io/f/xbjwdrql" method="post" class="contact-form">
+        <ul>
+          <li>
+            <input type="text" placeholder="Enter your Name" maxlength="30" required />
+          </li>
+          <li>
+            <input type="email" placeholder="Enter your email" required />
+          </li>
+          <li>
+            <textarea name="message" id="contact-message" cols="30" rows="7" maxlength="500"
```
Comment (mateo951) [inline index.html:193]: Fantastic job applying maxlength attribute 👌

File: index.html [hunk 5]
```diff
@@ -167,7 +167,39 @@ <h3 class="title">Skills</h3>
...
+              placeholder="Write your message here" required></textarea>
+          </li>
+          <li>
+            <button type="submit" class="submit-btn">Get in Touch</button>
+          </li>
+        </ul>
+      </form>
```
Comment (mateo951) [inline index.html:200]: All of your validations are on point 🔥🔥🔥

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi team, 

📝👌 **Your project highlights** 🔥😎 
- Linters are on point ✔
- Awesome PR 👌
- Nice Readme file 😎
- GitHub Flow is applied perfectly ✔

Congrats on your work guys! You have done a very good job implementing the form 🤯
There is one more adjustment to be made before you go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Review by julie-ify — APPROVED** [review]
## Hi team, 

# Highlights
- Good job implementing the required changes from the previous reviewer ✔️  

# Status: Approved ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

Approved emoji

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**